runtime/doc/lsp.txt

```diff
@@ -639,8 +639,8 @@ client()                                                      *vim.lsp.client*
                     automatically escalate and force shutdown.
                   • is_stopped() Checks whether a client is stopped. Returns:
                     true if the client is fully stopped.
-                  • on_attach(bufnr) Runs the on_attach function from the
-                    client's config if it was defined.
+                  • on_attach(client, bufnr) Runs the on_attach function from the
+                    client's config if it was defined. Useful for doing buffer-local setup.
```

Comment:
isn't it better without `doing` ?

Reply:
Yes (I just copied the verbage from the old lspconfig readme).

Reply:
Fixed